websites/E/e621/presence.ts

```diff
@@ -16,95 +16,83 @@ presence.on("UpdateData", async () => {
 		if (path === "/") {
 			search = document.querySelector("#tags");
 			if (search.value) {
-				presenceData.details = "Searching For:";
+				presenceData.details = "Searching for:";
+				presenceData.smallImageKey = "search";
 				presenceData.state = search.value;
 			} else presenceData.details = "Viewing the homepage";
 		} else if (path.includes("posts")) {
 			search = document.querySelector("#tags");
-			if (
-				search.value &&
-				search.value !==
-					document.querySelector<HTMLInputElement>("#tags_query").value
-			) {
-				presenceData.details = "Searching Posts For:";
+			mTitle = document
+				.querySelector<HTMLMetaElement>('meta[name="og:title"]')
+				?.content.replace(" - e926", "")
+				.replace(" ", "_");
```

Comment:
you're replacing every space with a _ just to split with a string with spaces later, didn't you mean to do it the other way around?